unicodetools/src/main/java/org/unicode/props/PropertyParsingInfo.java

```diff
@@ -1044,6 +1045,25 @@ private static void parseSimpleFieldFile(
                         setPropDefault(propInfo.property, defaultValue, "hardcoded", false);
                     }
                 }
+                if (line.getParts().length != 2) {
```

Comment:
optional: test for ` == 3`  ?

Reply:
Well, I really want to fail if it is anything other than 2 so we don’t miss it.

Reply:
I am ok with what you have, but you could be more specific:
```
if (3 parts and prop is Block) {
    // handle old Blocks.txt
    continue;
}
if (not 2 parts) {
    throw new ...
}
```